components/TranscriptHeader.tsx

```diff
@@ -9,12 +9,27 @@ type TranscriptHeaderParams = {
 };
 
 export default function TranscriptHeader({category, title, description, videoId} : TranscriptHeaderParams) {
+/*
+  const translationLinks = (
```

Comment:
Gonna incorporate that language list from before?

Reply:
eventually.

These are the top 5 languages in SPS.  

I'm just being lazy here with the comment.